src/component/form/TaggingElement/TaggingElement.tsx

```diff
@@ -33,32 +31,17 @@ export default class TaggingElement extends React.Component<TaggingElementProps,
     constructor(props: any) {
         super(props);
 
-        const tags = tagsToOptions(this.props.tags);
+        const tags = tagsToOptions(this.props.entry.value);
 
         this.state = {
-            tags,
-            errors: []
+            tags
         };
 
         bindCallbacks(this, {
             include: [/^handle/]
         });
     }
 
-    public validate(): boolean {
-        const errors: string[] = [];
-
-        if (this.props.required) {
-            if (this.state.tags.length === 0) {
-                errors.push(`${this.props.name} is required`);
-            }
-        }
-
-        this.setState({ errors });
-
-        return errors.length === 0;
-    }
-
```

Comment:
elements will no longer have anything to do with validation other than rendering validation status given to them